Sources/StreamChat/WebSocketClient/Events/Event.swift

```diff
@@ -30,6 +30,12 @@ protocol ChannelSpecificEvent: Event {
     var cid: ChannelId { get }
 }
 
+/// A bounding protocol for all events that have unread counts.
+public protocol HasUnreadCount: Event {
```

Comment:
The name "HasUnreadCount" feels slightly odd as a protocol, usually, this is on a boolean object.